src/features/offer/pages/Offer/Offer.tsx

```diff
@@ -36,9 +37,24 @@ export const Offer: FunctionComponent = () => {
     }
   })
 
+  const { data: offer } = useOffer({ offerId })
+  const similarOffers = useSimilarOffers(offerId, offer?.venue.coordinates)
+  const hasSimilarOffers = similarOffers && similarOffers.length > 0
+
+  const logSimilarOfferPlaylistVerticalScroll = useFunctionOnce(() => {
+    if (hasSimilarOffers) {
+      return analytics.logSimilarOfferPlaylistVerticalScroll()
+    }
+  })
+
   const { headerTransition, onScroll } = useOpacityTransition({
     listener: ({ nativeEvent }) => {
-      if (isCloseToBottom(nativeEvent)) logConsultWholeOffer()
+      if (isCloseToBottom(nativeEvent)) {
+        logConsultWholeOffer()
+      }
+      if (isCloseToBottom({ ...nativeEvent, padding: 300 })) {
```

Comment:
Peut être un petit commentaire pour préciser que l'event est déclenché lorsque la playlist s'affiche à l'écran

Reply:
[8d43d5b](https://github.com/pass-culture/pass-culture-app-native/pull/4111/commits/8d43d5bef5cd1b969a10a0a6a950ddc679299c44)

Reply:
`is triggered`:)